resources/certs/jwt/README.md

```diff
@@ -0,0 +1,97 @@
+
+Firstly, the public key literal can be supplied in the config.
+
+Second, a file containing the public key in the `pem` format can be placed in the this directory. The file name can then be specified to the config.
+
+The config can be supplied to the OpenHIM in the following ways:
+
+- Config File
+
+  - Public Key Filename.
+
+    ```json
+    // production.json
+    {
+      ...
+      "authentication": {
+        ...
+        "enableJWTAuthentication": true,
+        "jwt": {
+          "secretOrPublicKey": "<file-name>",
+          ...
+        }
+      }
+    }
+    ```
+
+  - Public Key Literal.
+
+    ```json
+    // production.json
+    {
+      ...
+      "authentication": {
+        ...
+        "enableJWTAuthentication": true,
+        "jwt": {
+          "secretOrPublicKey": "-----BEGIN PUBLIC KEY-----\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA4ZnfOESxGfb1MVD2coNy\n0G0bGarnKEz721MP30iyo6+YO3qzbETI8giIWGBtXD2VO49xk2miVIvZ3tAfPRnE\nsqJsOErfZ3ld5GrnLUSbUOr88cd+TTx4EqdU2dYAoc0iVEgA5UZJDLrWHM3VcHQl\nFc2F/JN78JBBZor2gWiABEFFShMN1PYmsx4IJUuE72gDVqblOLCfr+V1rT0C7iA1\n7V8lsm3jlRyBBNxdwqLvVXVcIip5/W5gQ/Ujq4KdXcC4LFR2J8idLEn4LPNsx6tA\ndHmAaBEHO9kyYgHijK+zi0b7qTYaPdrbM6siMFBh7HW6bobRqrFy5wR3zZuhg2Do\no8djtoJBHXNohxNm1D7iiNjH9jHSt9G2O5lAuDo19qb0jxMBR/ekQ3GZNTF3+C7z\nF6BXDPXY3S2Q7btYznQ6oTn/raqVaw4RiXDaBSotmOZHId2OnI5eNN4QTXr7RbOX\nzSqXf4OhiaW7Shjg0bbz6BUAKiMW6e0R3Z+JL8ZS47MxG6ibfimbZz9a0Hc5ItN2\nZK0mPPr2aEeLi/Wyaf7QB1N5IEZVj3YXJo/h5F37RnxV9IbRA201lKAVw5RbWo3Q\nur5jumzuID68U+i4rvB2JILlSykIrcGa7ffoMtTKJzTiHrKElBAEdgv4I1pUl0Js\ne2rYU8Kno94WC+34WGoUs7sCAwEAAQ==\n-----END PUBLIC KEY-----",
+          ...
+        }
+      }
+    }
+    ```
+
+    > Note: Use newline characters in the string to ensure the key structure.
+
+
+- Environment Variable
+
+  - Public Key Filename.
+
+    ```sh
+    authentication_jwt_secretOrPublicKey='<file-name>'
+    ```
+
+  - Public Key Literal.
+
+    ```sh
+    authentication_jwt_secretOrPublicKey='-----BEGIN PUBLIC KEY-----\nMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEA4ZnfOESxGfb1MVD2coNy\n0G0bGarnKEz721MP30iyo6+YO3qzbETI8giIWGBtXD2VO49xk2miVIvZ3tAfPRnE\nsqJsOErfZ3ld5GrnLUSbUOr88cd+TTx4EqdU2dYAoc0iVEgA5UZJDLrWHM3VcHQl\nFc2F/JN78JBBZor2gWiABEFFShMN1PYmsx4IJUuE72gDVqblOLCfr+V1rT0C7iA1\n7V8lsm3jlRyBBNxdwqLvVXVcIip5/W5gQ/Ujq4KdXcC4LFR2J8idLEn4LPNsx6tA\ndHmAaBEHO9kyYgHijK+zi0b7qTYaPdrbM6siMFBh7HW6bobRqrFy5wR3zZuhg2Do\no8djtoJBHXNohxNm1D7iiNjH9jHSt9G2O5lAuDo19qb0jxMBR/ekQ3GZNTF3+C7z\nF6BXDPXY3S2Q7btYznQ6oTn/raqVaw4RiXDaBSotmOZHId2OnI5eNN4QTXr7RbOX\nzSqXf4OhiaW7Shjg0bbz6BUAKiMW6e0R3Z+JL8ZS47MxG6ibfimbZz9a0Hc5ItN2\nZK0mPPr2aEeLi/Wyaf7QB1N5IEZVj3YXJo/h5F37RnxV9IbRA201lKAVw5RbWo3Q\nur5jumzuID68U+i4rvB2JILlSykIrcGa7ffoMtTKJzTiHrKElBAEdgv4I1pUl0Js\ne2rYU8Kno94WC+34WGoUs7sCAwEAAQ==\n-----END PUBLIC KEY-----'
+    ```
+
+    > Note: Use newline characters in the string to ensure the key structure.
+
+The Public Key file must be placed in this directory. i.e `openhim-core-js/resources/certs/jwt`
```

Comment:
Maybe a suggestion, but to move this line higher up when making mention to using a file instead of a string literal. It might get lots down here